plugins/modules/pip_package_info.py

```diff
@@ -27,6 +27,7 @@
     type: list
     elements: path
 requirements:
+  - pip >= 20.3b1 (necessary for the --format option)
```

Comment:
```suggestion
  - pip >= 20.3b1 (necessary for the C(--format) option)
```

Reply:
Sorry, I don't know the formatting commands from the docs. If that means "cursive", I approve :-)

Reply:
Ah, https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html . Monospace. Well then :-)